php-practice/section5.php

```diff
@@ -320,3 +338,25 @@ public function send() {
  *   'cost' => 3.99
  * ];
  */
+
+class ShoppingCart {
+  private $item;
+  private $shoppingCart;
+
+  $item = [
+    'name' => 'Pringles',
+    'cost' => 3.99
+  ];
```

Comment:
These 4 lines *should* cause PHP to blow up. You're just inside of the `class` block -- you're not inside of a function.

If you're trying to add a property to this class, you have to set the visibility, e.g.:

```PHP
private $item;
public $item;
```

If you're trying to add a property **and** give it a default value, you can set the value in the same place where the property is declared:

```PHP
private $item = [
  'name' => 'Pringles'
];

public $foo = 'BAR';
public $baz = 123;
```